specs/ingress/tests.go

```diff
@@ -3,105 +3,145 @@ package ingress
 import (
 	"fmt"
 	"net/http"
-	"strings"
+	"os/exec"
 	"time"
 
 	"github.com/linkerd/linkerd2-conformance/utils"
-	"github.com/onsi/ginkgo"
 	"github.com/onsi/gomega"
 )
 
-func pingEmojivoto(ip string) error {
-	req, err := http.NewRequest("GET", fmt.Sprintf("http://%s", ip), nil)
+func pingIngress() error {
```

Comment:
NIt: Consider renaming this function, because in the context of network tests, `ping` means ICMP to me. How about just `httpGet(url string)`?